{
	"id": "CVE-2018-12175",
	"vulnerabilities": [
		{
			"content": {
				"id": "CVE-2018-12175",
				"description": "Default install directory permissions in Intel Distribution for Python (IDP) version 2018 may allow an unprivileged user to escalate privileges via local access.",
				"severity": [
					{
						"type": "cvss_v2",
						"source": "nvd@nist.gov",
						"cvss_v2": {
							"vector": "AV:L/AC:L/Au:N/C:P/I:P/A:P",
							"base_score": 4.6,
							"nvd_base_severity": "MEDIUM",
							"temporal_score": 4.6,
							"nvd_temporal_severity": "MEDIUM",
							"environmental_score": 4.6,
							"nvd_environmental_severity": "MEDIUM"
						}
					},
					{
						"type": "cvss_v30",
						"source": "nvd@nist.gov",
						"cvss_v30": {
							"vector": "CVSS:3.0/AV:L/AC:L/PR:L/UI:N/S:U/C:H/I:H/A:H",
							"base_score": 7.8,
							"base_severity": "HIGH",
							"temporal_score": 7.8,
							"temporal_severity": "HIGH",
							"environmental_score": 7.8,
							"environmental_severity": "HIGH"
						}
					}
				],
				"cwe": [
					{
						"source": "nvd@nist.gov",
						"cwe": [
							"CWE-276"
						]
					}
				],
				"references": [
					{
						"source": "af854a3a-2127-422b-91ae-364da2661108",
						"url": "https://www.intel.com/content/www/us/en/security-center/advisory/intel-sa-00181.html"
					},
					{
						"source": "nvd.nist.gov",
						"url": "https://nvd.nist.gov/vuln/detail/CVE-2018-12175"
					},
					{
						"source": "secure@intel.com",
						"url": "https://www.intel.com/content/www/us/en/security-center/advisory/intel-sa-00181.html"
					}
				],
				"published": "2018-09-12T19:29:02.107Z",
				"modified": "2024-11-21T03:44:42.017Z"
			},
			"segments": [
				{
					"ecosystem": "cpe"
				}
			]
		}
	],
	"detections": [
		{
			"ecosystem": "cpe",
			"conditions": [
				{
					"criteria": {
						"operator": "OR",
						"criterias": [
							{
								"operator": "OR",
								"criterias": [
									{
										"operator": "OR",
										"criterias": [
											{
												"operator": "OR",
												"criterions": [
													{
														"type": "version",
														"version": {
															"vulnerable": true,
															"fix_status": {
																"class": "unknown"
															},
															"package": {
																"type": "cpe",
																"cpe": "cpe:2.3:a:intel:distribution_for_python:2018:*:*:*:*:*:*:*"
															}
														}
													}
												]
											}
										]
									}
								]
							}
						]
					}
				}
			]
		}
	],
	"data_source": {
		"id": "nvd-api-cve",
		"raws": [
			"vuls-data-raw-nvd-api-cve/2018/CVE-2018-12175.json"
		]
	}
}
